Week-11/redux-1/src/app/store.js

```diff
@@ -0,0 +1,9 @@
+import { configureStore } from "@reduxjs/toolkit";
+
+import switchReducer from "features/switch/switchSlice";
+
+export const store = configureStore({
+  reducer: {
+    theme: switchReducer,
```

Comment:
Can we have a rootReducer? So that in the future if we have multiple reducers we dont have to worry about changing the import statement here.